etc/Dockerfile_build

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:trusty
+FROM ubuntu:latest
```

Comment:
try this will `rolling` or lock it into `bionic` so that it doesnt shift to something too shiny

Reply:
now with about 10% less shininess!